src/ripgrep-directory-searcher.js

```diff
@@ -146,8 +150,16 @@ function processSubmatch (submatch, lineText, offsetRow) {
 
   return {
     range: [start, end],
-    lineText: cleanResultLine(lineParts.join('\n'))
+    lineText: cleanResultLine({ text: lineParts.join('\n') })
+  }
+}
+
+function getText (input) {
+  if (input.text) {
+    return input.text
   }
+
+  return Buffer.from(input.bytes, 'base64').toString()
```

Comment:
Minor: Putting this in an `else` block would be more readable to me.

Reply:
I've just used a ternary at the end 😁